tests/compas/scene/test_scene.py

```diff
@@ -83,22 +83,19 @@ def test_sceneobject_auto_context_discovery_no_context(mocker):
     def test_sceneobject_transform():
         scene = Scene()
         sceneobj1 = scene.add(Box())
-        sceneobj1.frame = Frame([1.0, 0.0, 0.0], xaxis=[1.0, 0.0, 0.0], yaxis=[0.0, 1.0, 0.0])
         sceneobj1.transformation = Translation.from_vector([10.0, 0.0, 0.0])
-        assert sceneobj1.worldtransformation == sceneobj1.frame.to_transformation() * sceneobj1.transformation
+        assert sceneobj1.worldtransformation == Translation.from_vector([10.0, 0.0, 0.0])
+        assert sceneobj1.frame == Frame([10.0, 0.0, 0.0], [1.0, 0.0, 0.0], [0.0, 1.0, 0.0])
 
         sceneobj2 = scene.add(Box(), parent=sceneobj1)
-        sceneobj2.frame = Frame([1.0, 1.0, 0.0], xaxis=[1.0, 0.0, 0.0], yaxis=[0.0, 1.0, 0.0])
```

Comment:
by simply removing the frame, you're not reproducing the same result as before...

Reply:
Yep, but if you see in next few lines, the assert condition was updated according to the new computing scheme, or perhaps I misunderstood something here?